CMakeLists.txt

```diff
@@ -1978,9 +1955,9 @@ endif()
 # Configure Warnings
 if(HPX_WITH_COMPILER_WARNINGS)
   if(MSVC) # Adding special warning settings for the MSVC compiler ...
-    hpx_add_compile_flag(-W3 LANGUAGES C CXX)
+    hpx_add_compile_flag(-W3 LANGUAGES CXX)
     # According to the ifort Windows manual, W3 isn't supported
-    hpx_add_compile_flag(-W1 LANGUAGES Fortran)
+    hpx_add_compile_flag(-W1 LANGUAGES)
```

Comment:
These two lines can go

Reply:
I've at the same time removed the `LANGUAGES CXX` arguments since that's the default. I've kept the option though, in case we need it in the future again.